.vscode/launch.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
IDE settings should not be in the pr

Reply:
@VincentLanglet
Thanks for your review. I will revise them.